packages/controls-navigator/src/NavigatorControl.vue

```diff
@@ -0,0 +1,293 @@
+<template>
+  <div :class="getNavigateClass()">
+    <v-stepper :model-value="currentStep - 1">
+      <template v-for="index in steps" :key="index">
+        <v-stepper-item
+          :editable="navigationType === 'tab' || currentStep > index"
+          :title="prompts[index - 1] ? prompts[index - 1].name : ''"
+          :complete="getComplete(index)"
+          :class="getStepClass(currentStep, index)"
+          @click="gotoStep(index)"
+        />
+        <div class="breadcrumbs-container">
+          <YOUIBreadcrumbs
```

Comment:
As mentioned before, this component name `YOUIBreadcrumbs` should be updated to something like `BreadcrumbsContainer` and not mention yeoman ui

Reply:
has changed it.